scss/lessonsChallenges.scss

```diff
@@ -0,0 +1,64 @@
+.default_challenge_status{
+  background-color: grey;
+}
+
+.passed_challenge_status{
+  background-color: #00cf92;
+}
+
+.unapproved_challenge_status{
+  background-color: red;
+}
+
+.pending_challenge_status{
+ background-color: orange;
+}
+
+.lesson_challenges{
+  display: flex;
+  margin-top: 15px
+}
+
+.lessons_challenges_title{
```

Comment:
there are a few of these, perhaps making these `.profile_section_title` would be best, then you don't have to repeat `padding: 8px` css over and over again.